spec/support/initializers/capybara.rb

```diff
@@ -2,7 +2,7 @@
 require "capybara/rspec"
 require "webdrivers/chromedriver"
 
-Webdrivers.cache_time = 86_400
+Webdrivers.cache_time = 0
```

Comment:
i actually don't think that this does anything 😩 according to [these docs](https://github.com/titusfortner/webdrivers#caching-drivers), i think it's actually how we tell the webdrivers to update itself/look for updates?

Reply:
Yeah, that's just about self update